admin/actions.php

```diff
@@ -653,3 +653,11 @@ function ajax_cron_test() {
 
 }
 add_action( 'wp_ajax_hmbkp_cron_test', 'HM\BackUpWordPress\ajax_cron_test' );
+
+/**
+ * Remember notice dismissal
+ */
+function hmbkp_dismiss_notice() {
```

Comment:
Happy for that to be a separate issue, perhaps even part of https://github.com/humanmade/backupwordpress/issues/762